### ISSUE TITLE
Add WinPython 3.4.4.4 to Release 2016-04

### ISSUE THREAD

**Issue by hiccup7** [winpython/winpython#393]
mingwpy doesn't support Python 3.5:
https://github.com/mingwpy/mingwpy/issues/25#issuecomment-239698119

The current release of matplotlib is incompatible with Qt5:
https://groups.google.com/forum/#!topic/winpython/8G9SiojkBII

Therefore, Python 3.4 with Qt4 would be good to continue to maintain until at least one of the above issues is resolved.  I suggest to add WinPython 3.4.4.4 (Qt4) to Release 2016-04.

**Comment by stonebig** [comment]
Mingwpy and Qt problems are unrelated, for me.

Matplotlib 1.5.3 or 2.0.0b4 from yesterday, should have the issue fixed:
- `pip uninstall Matplotlib`
- `pip install Matplotlib`

if you want Qt4, you may try this:
- download related wheel from cgohlke in d;\toto
- start winpython command prompt
- `wppm -u PyQt5`
- `pip install --no-index --find-links=d:\toto PyQt4`

**Comment by stonebig** [comment]
PyQt4 is not maintained anymore, and nowodays this means that it must be removed from refresh of Python distros:
- before it bites,
- before it slows down other packages forced to maintain compatibility.

As users can't change their code/practice overnight, I remove PyQt4 progressively.

**Comment by hiccup7** [comment]
The Qt5+matplotlib issue was solved after I submitted this issue.  That is good news.

My PC with WinPython has no internet connection.  I have been using the WinPython Control Panel to uninstall and install packages.  I always use the wheels from cgohlke when available.

I have been staying with Qt4 until Spyder 3 is released because Qt5 requires a beta (buggy) version of Spyder.  Spyder is one of my most important packages.  Only the released version (2.3.9) wheel is available from @cgohlke.  Spyder doesn't show up in a search at PyPI.  The Spyder project releases page has no wheels or executable installers for the beta versions.

In summary, offline WinPython users wanting Cython/mingwpy without having to buy MS Visual Studio need Python 3.4 (for the next 3 years?).  WinPython users wanting a stable version and good experience with Spyder need Qt4 (at present).

**Comment by stonebig** [comment]
@cgholke will likely drop support for Python3.4 in Q1 2017, shifting to Python 3.6.
==> That is an un-avoidable death-line for WinPython3.4 maintenance, be prepared.

I understand the remark that https://aka.ms/vcpython35 is not installable without internet.

Would a link to a Spyder3.0.0b6 "wheel" version, or today master fit you ?

**Comment by hiccup7** [comment]
Thank you for providing wheels for the latest Spyder beta.  Very generous.

Users of WinPython release 2016-04 with Qt5 will likely want to update matplotlib and the beta Spyder.  Would you like to post the Spyder wheels to the WinPython Google group?

If a Qt5 user finds a beta Spyder bug intolerable, they have the option of downgrading to Qt4 and Spyder 2.3.9.  I have already done this successfully for Spyder with the WinPython Control Panel.  Can Qt4 also be downgraded this way?

What if a beta Spyder bug is intolerable and it is fixed in the master branch and then beta 7?  Is Spyder pure Python?  If so, can we download the master branch as a ZIP file and use it in WinPython Control Panel?

**Comment by stonebig** [comment]
hum, wppm code to remove PyQt5.5 "binary package" is buggy.

This last binary package will annoy me till the end...

**Comment by stonebig** [comment]
WinPython-3.4 efforts do stop there:
- the legacy compiler vs2010 used for Python-3.4 is not C++99 compatible for Qt5.7 and friends,
- the mingwpy C++99 alternative compiler never crossed beta3 status and established support, 
- only a binary Qt5 installer is available, forcing WinPython to keep legacy stuff.

WinPython-3.6 arrives earlier then expected.

**Comment by hiccup7** [comment]
Note that many WinPython scripts (such as env.bat) look for the existence of the folder "%WINPYDIR%\Lib\site-packages\PyQt5" to determine if Qt5 is used.  For WinPython Control Panel and other WinPython scripts to work after the procedure above, it is necessary to manually delete this folder.
